input/resources/au-patient.xml

```diff
@@ -104,20 +111,23 @@
       <path value="Patient.extension" />
       <sliceName value="genderIdentity" />
       <short value="The patient's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
```

Comment:
Why say "the patient's". Too long a text. We don't do that for other invariants. Just say Gender Identity.